conf/broker.conf

```diff
@@ -624,6 +624,9 @@ bookkeeperClientTimeoutInSeconds=30
 # Using a value of 0, is disabling the speculative reads
 bookkeeperClientSpeculativeReadTimeoutInMillis=0
 
+# Number of channels per bookie
+bookkeeperNumberOfChannelsPerBookie=16
```

Comment:
I am not sure 16 is a good default value. Can't we just set the default value to 1?

Reply:
right now, hardcoded value is `16` and it was configured > 1 due to per bookie cnx bottleneck. So, changing it to 1 will create an issue for existing system if they will try to use upgrade release without knowledge of this configuration. However, recently while doing testing on high throughput bookies, we figured out that 16 bookie channel was the bottleneck and we need to make it configurable to tune it based on number topics and per bookie traffic.